DocumentedExamples/NearestNeighbourDistance.ipynb

```diff
@@ -0,0 +1,1730 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    x, y = np.meshgrid(np.radians(var_ice.xt_ocean.values),
</pre><p> </p><ol><li>Shall we use <code>np.deg2rad</code> instead of <code>np.radians</code>?</li><li>  <code>x, y</code> are angles in radians and not distances... Isn't <code>x, y</code> confusing? Perhaps <code>lon_radians, lat_radians</code>  ?  </li></ol><p> </p><p>x, y are radians?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>